apps/crn-server/src/routes/news.route.ts

```diff
@@ -9,9 +10,14 @@ export const newsRouteFactory = (newsController: NewsController): Router => {
   newsRoutes.get('/news', async (req, res) => {
     const { query } = req;
 
-    const options = validateFetchPaginationOptions(query);
+    const options = validateFetchOptions(query);
 
-    const result = await newsController.fetch(options);
+    const result = await newsController.fetch({
+      ...options,
+      filter: options.filter && {
+        frequency: options.filter as NewsFrequency[],
```

Comment:
ok so basically what TS is telling us is that we are trying to assign a string into a union, as far as I can tell.

What I propose you should do is to change the validation to be more specific ie use a dedicated validation function (like we for many other routes) then change the validation type and implementation to accept an enum instead of string.